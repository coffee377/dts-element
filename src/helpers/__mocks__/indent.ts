export const indent = (line: string): string => `[indent ${line}]`;
