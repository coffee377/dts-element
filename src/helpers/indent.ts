export const indent = (str: string): string => str
  .split('\n')
  .map((line: string) => `  ${line}`)
  .join('\n');
