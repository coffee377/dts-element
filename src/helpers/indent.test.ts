import { indent } from './indent';

it('should return indented string', () => {
  expect(indent('line 1\nline 2\nline 3')).toMatchSnapshot();
});
