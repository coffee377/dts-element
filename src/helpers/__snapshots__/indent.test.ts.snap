// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return indented string 1`] = `
"  line 1
  line 2
  line 3"
`;
